scripts/nats-bootstrap/README.md

```diff
@@ -0,0 +1,62 @@
+### NATS auth terminology
+
+- `Accounts` group users and Jetstream support is enabled/disabled at the account level NOTE: pub/sub permissions set at the Account level are not inherited by users of that account.
+- `Signing keys` are associated with accounts and can be 'templated' with pub/sub permissions and have a `role` name set.
+- `Users` are applications connecting to NATS to pub/sub, users can be assigned a `role`G.
+
+## Nats accounts configuration
```

Comment:
I think this could be expanded upon. It took me a couple of read-throughs to understand that `new instance` actually meant "re-creating a sandbox using (the same?) instance of NATS you set up previously.

I'll see if I can't suggest something going forward.